app/viewmodels/helpers/events/EventsHelper.scala

```diff
@@ -69,6 +70,16 @@ class EventsHelper @Inject()(
     )
   }
 
+  private def ie802ChangeDestinationDueHtml(event: MovementHistoryEvent)(implicit messages: Messages): Html =
+    HtmlFormat.fill(
+      Seq(
+        p(classes = "govuk-body-l")(Html(
+          messages(s"${timelineHelper.getEventBaseKey(event)}.p1", event.sequenceNumber)
```

Comment:
We don't need to pass sequence number to the message as there is no placeholder for it within the content

Reply:
Well spotted, that'll teach me for copying it from above 😂